torchtitan/utils.py

```diff
@@ -52,18 +52,19 @@ def _warn_overwrite_env(env, val):
     os.environ[env] = val
 
 
-def set_determinism(seed: Optional[int]) -> None:
+def set_determinism(seed: Optional[int], deterministic: bool) -> None:
```

Comment:
The function name is slightly confusing now. I wonder if we can separate `set_seed` and `set_determinism` into two functions.